src/main/java/frc/trigon/robot/subsystems/differentialDrive/DifferentialDriveConstants.java

```diff
@@ -0,0 +1,37 @@
+package frc.trigon.robot.subsystems.differentialDrive;
+
+import com.ctre.phoenixpro.configs.TalonFXConfiguration;
+import com.ctre.phoenixpro.hardware.TalonFX;
+import com.ctre.phoenixpro.signals.InvertedValue;
+import com.ctre.phoenixpro.signals.NeutralModeValue;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+
+public class DifferentialDriveConstants {
+    private static final int RIGHT_FRONT_MOTOR_ID = 0;
+    private static final int RIGHT_REAR_MOTOR_ID = 1;
+    private static final int LEFT_FRONT_MOTOR_ID = 2;
+    private static final int LEFT_REAR_MOTOR_ID = 3;
+    private static final InvertedValue INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Coast;
+    private static final TalonFX RIGHT_FRONT_MOTOR = new TalonFX(RIGHT_FRONT_MOTOR_ID);
+    private static final TalonFX RIGHT_REAR_MOTOR = new TalonFX(RIGHT_REAR_MOTOR_ID);
+    private static final TalonFX LEFT_FRONT_MOTOR = new TalonFX(LEFT_FRONT_MOTOR_ID);
+    private static final TalonFX LEFT_REAR_MOTOR = new TalonFX(LEFT_REAR_MOTOR_ID);
+    private static final MotorControllerGroup RIGHT_MOTORS = new MotorControllerGroup(RIGHT_FRONT_MOTOR, RIGHT_REAR_MOTOR);
+    private static final MotorControllerGroup LEFT_MOTORS = new MotorControllerGroup(LEFT_FRONT_MOTOR, LEFT_REAR_MOTOR);
```

Comment:
No need for the repetetive  `private static final int`, `private static final TalonFX` and `private static final MotorControllerGroup`